cargo-audit/src/commands/audit.rs

```diff
@@ -1,5 +1,7 @@
 //! The `cargo audit` subcommand
 
+#![allow(unused_qualifications)]
```

Comment:
Rust 1.70.0 produces:

```
warning: unnecessary qualification
  --> cargo-audit/src/commands/audit.rs:38:12
   |
38 |     #[clap(subcommand)]
   |            ^^^^^^^^^^
   |
note: the lint level is defined here
  --> cargo-audit/src/lib.rs:21:56
   |
21 | #![warn(missing_docs, rust_2018_idioms, trivial_casts, unused_qualifications)]
   |                                                        ^^^^^^^^^^^^^^^^^^^^^
```

I did not investigate it further for now.

Reply:
I really need to upgrade `clap` (and `abscissa`)